spec/fixture/config/settings.yml

```diff
@@ -5,3 +5,4 @@ array:
     - test1
 object:
     test1: value
+falsey: false
```

Comment:
it's spelled `falsey` in [rspec](https://relishapp.com/rspec/rspec-expectations/v/3-5/docs/built-in-matchers) and `falsy` in [jasmine](https://jasmine.github.io/2.0/introduction.html) :D